# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/telemetry/Histograms.json
# Commit: a7c322ebcfea
# Full Hash: a7c322ebcfea97985f4ae7708cf94793e89d255e
# Author: Brad Werth <bwerth@mozilla.com>
# Date: 2021-12-18 09:43:08
# Regressor Bug: 1737682
# File Overlap Count: 1
# Description:
#   Bug 1737682 Part 2: Log whether or not we are hitting the detached state. r=mstange
#   
#   There are 13 enum cases that we track internally, 2 of which aren't emitted
#   as telemetry. These cases are 0-indexed to match the enum values.
#   
# ==============================================================================

diff -r 692b96771caf -r a7c322ebcfea toolkit/components/telemetry/Histograms.json
--- a/toolkit/components/telemetry/Histograms.json	Fri Dec 17 20:48:59 2021 +0000
+++ b/toolkit/components/telemetry/Histograms.json	Fri Dec 17 21:14:24 2021 +0000
@@ -15965,6 +15965,17 @@
     "description": "The amount of async paint tasks queued to the paint thread during a layer transaction.",
     "bug_numbers": [1483245]
   },
+  "GFX_MACOS_VIDEO_LOW_POWER": {
+    "record_in_processes": ["main"],
+    "products": ["firefox"],
+    "alert_emails": ["bwerth@mozilla.com"],
+    "expires_in_version": "never",
+    "releaseChannelCollection": "opt-out",
+    "bug_numbers": [1737682],
+    "description": "MacOS video low power state achieved when enqueueing a video frame.",
+    "kind": "categorical",
+    "labels": ["None", "FailNotVideo", "Active", "FailPref", "Fail10_13", "FailFullscreen", "FailIsoMouse", "FailIsoTopVideo", "FailIsoSize", "FailIsoCenter", "FailIsoOneVideo", "FailSurface", "FailEnqueue"]
+  },
   "PERMISSION_REQUEST_ORIGIN_SCHEME": {
     "record_in_processes": ["main"],
     "products": ["firefox", "fennec"],
